frontend/src/components/Client/Skeletons/PopularProductSkeleton/PopularProductSkeleton.js

```diff
@@ -0,0 +1,33 @@
+import Skeleton from 'react-loading-skeleton'
+import 'react-loading-skeleton/dist/skeleton.css'
+
+const PopularProductSkeleton = () => {
```

Comment:
I think this needs some work -
We want to try aligning the elements of the card to the text as much as possible. It doesn't have to match 100% but should give the illusion that something is loading and appearing in somewhat the same spot. I'll attach a picture of the current skeleton with the fully loaded card on top of it. Theres some discrepancies that appear in the skeleton vs the loaded card. I'm okay with the title, price and card button being slightly off, but the description is way off here. I'll also attach what I think it should look like to better match the card dimensions-
![image](https://user-images.githubusercontent.com/5695484/229263937-c0a6bd37-2e35-445e-8765-a504d6aab86f.png)